__tests__/app.test.js

```diff
@@ -25,4 +25,23 @@ describe.only('GET api/topics', () => {
             }))
         })
     });
+});
+
+describe('GET api/articles/:article_id', () => {
+    it('should return an article object containing the correct properties', () => {
+        const ID = 3
+        return request(app).get(`/api/articles/${ID}`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual({
```

Comment:
I would like this toEqual to instead be toMatchObject, when we add comment_count later, using toEqual will fail this test